python/__init__.py

```diff
@@ -16,6 +16,33 @@ def get_version():
         return version_file.read().strip()
 
 
+def modcc():
+    import os
+    import sys
+    import subprocess
+
+    sys.exit(
+        subprocess.call(
+            [os.path.join(os.path.dirname(__file__), "bin", "modcc"), *sys.argv[1:]]
+        )
+    )
+
+
+def build_catalogue():
+    import os
+    import sys
+    import subprocess
+
+    sys.exit(
+        subprocess.call(
+            [
+                os.path.join(os.path.dirname(__file__), "bin", "arbor-build-catalogue"),
+                *sys.argv[1:],
+            ]
+        )
+    )
+
+
```

Comment:
PS: I lied; one more. If we start adding Python-only functions to Arbor (all for it), let's make a separate module and just use `__init__.py` for importing all of the sub-modules.

Reply:
Since this is a workaround for functionality not meant for direct use, I don't think this is necessary yet.